### PR TITLE
CMD docker build file step fix

### PR DESCRIPTION
**Pull request by lotharschulz**
CMD as one line and closing square bracket added

### CODE REVIEW

**Comment by lotharschulz** [comment]
@rafaelcaricio : pls check

File: Dockerfile [hunk 1]
```diff
@@ -52,7 +52,4 @@ RUN \
   touch /var/log/ghe-delete-instuck-progress.log && \
   chown -R application: /var/log/ghe-delete-instuck-progress.log
 
-CMD ["python3", "/delete-instuck-backups/delete_instuck_progress.py",
-    "/kms/convert-kms-private-ssh-key.sh",
-    "cron",
-    "tail -F /var/log/ghe-prod-backup.log"
+CMD ["python3", "/delete-instuck-backups/delete_instuck_progress.py", "/kms/convert-kms-private-ssh-key.sh", "cron", "tail -F /var/log/ghe-prod-backup.log"]
```
Comment (rafaelcaricio) [inline Dockerfile]: Is it all a single command?

Comment (lotharschulz) [inline Dockerfile]: intended are 4 commands

File: Dockerfile [hunk 2]
```diff
@@ -45,14 +48,12 @@ RUN \
...
 # delete_instuck_progress log
 RUN \
   touch /var/log/ghe-delete-instuck-progress.log && \
   chown -R application: /var/log/ghe-delete-instuck-progress.log
 
-CMD ["python3", "/delete-instuck-backups/delete_instuck_progress.py",
-    "/kms/convert-kms-private-ssh-key.sh",
-    "cron",
-    "tail -F /var/log/ghe-prod-backup.log"
+CMD ["/bin/bash", "/backup/final-docker-cmd.sh"]
```
Comment (rafaelcaricio) [inline Dockerfile]: I think you do not need this part `"/bin/bash",`

Comment (lotharschulz) [inline Dockerfile]: true, idea was to be explicit

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**